pkg/controller/util.go

```diff
@@ -121,13 +120,13 @@ func (c *Controller) getSecretName(pvc *v1.PersistentVolumeClaim) (string, error
 		} else {
 			msg += "secret name is missing from annotation %q in pvc \"%s/%s\""
 		}
-		glog.Infof(msg+"\n", AnnSecret, ns, pvc.Name)
+		glog.V(Vadmin).Infof(msg+"\n", AnnSecret, ns, pvc.Name)
 		return "", nil // importer pod will not contain secret credentials
 	}
-	glog.Infof("getEndpointSecret: retrieving Secret \"%s/%s\"\n", ns, name)
+	glog.V(Vdebug).Infof("getEndpointSecret: retrieving Secret \"%s/%s\"\n", ns, name)
```

Comment:
Admin's may like to see that a supplied secret was being processed

Reply:
yes, but we don't know if the Get will succeed. I'll keep this as a debug msg and add a user level msg.